generator/resources/eks_e2e_jmx_test_matrix.json

```diff
@@ -0,0 +1,5 @@
+[
+  {
+    "nodes": 1
```

Comment:
Does this imply just 1 node in the cluster ? Why just 1 ?

Reply:
Yeah, we can re-configure this, but I just went for 1 since that seemed sufficient for these JMX tests. I can change the number if you think it's needed.